### PR TITLE
SEO-175680-Typscript-Grid-controls-redirects-link

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: TypeScript/Grid/Editing.md [hunk 1]
```diff
@@ -768,7 +768,7 @@ After the template elements are converted to JS controls using actionComplete ev
 
 Users can start editing by clicking a cell and typing data into it. Edited cell will be marked while navigating to next cell or any other row, so that you know which fields or cells has been edited. Set [`editMode`](https://help.syncfusion.com/api/js/ejgrid#members:editsettings-editmode "editMode") as `batch` to enable batch editing.
 
-N> Refer the KB [link](http://www.syncfusion.com/kb/3016/how-to-suppress-grid-confirmation-messages# "link") for "How to suppress grid confirmation messages" in batch mode.
+N> Refer the KB [link](https://support.syncfusion.com/kb/article/3065/how-to-suppress-grid-confirmation-messages?isInternalRefresh=False "link") for "How to suppress grid confirmation messages" in batch mode.
```
Comment (Yvone-Atieno) [inline TypeScript/Grid/Editing.md:771]: Check this link
image

File: TypeScript/Grid/Editing.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The grid control has support for the dynamic insertion, updating and deletion of
...
+The primary key for the data source should be defined in [`columns`](https://help.syncfusion.com/api/js/ejgrid#members:columns "columns") definition, for editing to work properly. In [`columns`](https://help.syncfusion.com/api/js/ejgrid#members:columns "columns") definition, particular primary column's [`isPrimaryKey`](https://help.syncfusion.com/api/js/ejgrid#members:columns-isprimarykey "isPrimaryKey") property should be set to `true`. Refer to the Knowledge base [link](https://support.syncfusion.com/kb/article/2934/cant-edit-any-row-except-the-first-row-in-grid?isInternalRefresh=False "link") for more information.
```
Comment (Yvone-Atieno) [inline TypeScript/Grid/Editing.md:15]: Check KB link

File: TypeScript/Grid/Data-Binding.md [hunk 3]
```diff
@@ -51,15 +51,15 @@ The following output is displayed as a result of the above code example.
...
+To bind remote data to Grid Control, you can assign a service data as an instance of [`ej.DataManager`](https://help.syncfusion.com/api/js/ejdatamanager "DataManager") to the [`dataSource`](https://help.syncfusion.com/api/js/ejgrid#members:datasource "dataSource") property.
 
 ### OData
 
-OData is a standardized protocol for creating and consuming data. You can provide the [OData service](https://www.odata.org/) URL directly to the [`ej.DataManager`](https://help.syncfusion.com/api/js/ejdatamanager# "DataManager") class and then you can assign it to Grid [`dataSource`](https://help.syncfusion.com/api/js/ejgrid#members:datasource "datasource").
+OData is a standardized protocol for creating and consuming data. You can provide the [OData service](https://www.odata.org) URL directly to the [`ej.DataManager`](https://help.syncfusion.com/api/js/ejdatamanager "DataManager") class and then you can assign it to Grid [`dataSource`](https://help.syncfusion.com/api/js/ejgrid#members:datasource "datasource").
```
Comment (Yvone-Atieno) [inline TypeScript/Grid/Data-Binding.md:62]: Revert OData service link

File: TypeScript/Grid/Data-Binding.md [hunk 4]
```diff
@@ -51,15 +51,15 @@ The following output is displayed as a result of the above code example.
 
 
 N> 1. There is no in-built support to bind the XML data to the grid. But you can achieve this requirement with the help of [custom adaptor](https://help.syncfusion.com/js/datamanager/data-adaptors#custom-adaptor) concept. 
-N> 2. Refer this [Knowledge Base link](https://www.syncfusion.com/kb/3377/how-to-process-xml-data-from-server-using-datamanager-and-bound-to-grid) for bounding XML data to grid using custom adaptor. 
+N> 2. Refer this [Knowledge Base link](https://support.syncfusion.com/kb/article/3562/how-to-process-xml-data-from-server-using-datamanager-and-bound-to-aspnet-mvc-grid?isInternalRefresh=False) for bounding XML data to grid using custom adaptor.
```
Comment (Yvone-Atieno) [inline TypeScript/Grid/Data-Binding.md:54]: Check KB link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: TypeScript/Grid/Editing.md (hunk 2 above)
Comment (Edithabich) [reply TypeScript/Grid/Editing.md:15]: Resolved

File: TypeScript/Grid/Data-Binding.md (hunk 4 above)
Comment (Edithabich) [reply TypeScript/Grid/Data-Binding.md:54]: Resolved

File: TypeScript/Grid/Data-Binding.md (hunk 3 above)
Comment (Edithabich) [reply TypeScript/Grid/Data-Binding.md:62]: Resolved

File: TypeScript/Grid/Editing.md (hunk 1 above)
Comment (Edithabich) [reply TypeScript/Grid/Editing.md:771]: Resolved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**